packages/config/config/devices/0x027a/zen04.json

```diff
@@ -0,0 +1,114 @@
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN04",
+	"description": "Indoor Smart Plug",
+	"devices": [
+		{
+			"productType": "0x7000",
+			"productId": "0xb002"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "LED Indicator Mode",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Follow outlet",
+					"value": 0
+				},
+				{
+					"label": "Opposite of outlet",
+					"value": 1
+				},
+				{
+					"label": "Always off",
+					"value": 2
+				},
+				{
+					"label": "Always on",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "2",
+			"$import": "templates/zooz_template.json#auto_off_timer_duration",
+			"label": "Auto Turn-Off Timer Duration"
+		},
+		{
+			"#": "3",
+			"$import": "templates/zooz_template.json#auto_on_timer_duration",
+			"label": "Auto Turn-On Timer Duration"
+		},
+		{
+			"#": "4",
+			"label": "On/Off Status Recovery After Power Failure",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Previous status",
+					"value": 0
+				},
+				{
+					"label": "Always off",
+					"value": 1
+				},
+				{
+					"label": "Always on",
+					"value": 2
+				}
+			]
+		},
```

Comment:
This should use a template like the ZEN05 does here:
https://github.com/zwave-js/node-zwave-js/blob/81d6306c505b8af4d51ba8a10e645c6a8dd0f7a7/packages/config/config/devices/0x027a/zen05.json#L48-L51
except that you need the `..._prev_off_on` variant.